### PR TITLE
Add makefile

### PR DESCRIPTION
**Pull request by eclimov**
- Added makefile

This is a very small, but [in my opinion] valuable, addition to this awesome tutorial.

### CODE REVIEW

**Comment by gregurco** [comment]
@eclimov thank you for contribution :+1: Could you please propose changes to branch `day_1` and not master, because the history starts not from `master` here.

File: makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+stop:
+	docker-compose stop
+
+php:
+	docker-compose exec php-fpm sh
+
+mysql:
+	docker-compose exec mysql sh
+
+mailhog:
```
Comment (gregurco) [inline makefile:13]: Did you had cases when you had to enter in this container?

Comment (eclimov) [reply makefile:13]: No. Looks like it is redundant, indeed.

**Comment by eclimov** [comment]
> @eclimov thank you for contribution 👍 Could you please propose changes to branch `day_1` and not master, because the history starts not from `master` here.

Sure. Added #14 
Closing this one.

**Pull request closed without merging**